### PR TITLE
Change logo for xmas :christmas_tree: 

### PR DESCRIPTION
**Pull request by nicolaswurtz**

### CODE REVIEW

File: front/public/pouet.png
Comment (flomonster) [inline front/public/pouet.png:1]: Why this name?

**Review by flomonster — CHANGES_REQUESTED** [review]

File: front/public/pouet.png
Comment (nicolaswurtz) [reply front/public/pouet.png:1]: sorry, I fix up

**Review by flomonster — APPROVED** [review]
LGTM :christmas_tree:

**Review by Math-R — APPROVED** [review]

**Review by anisometropie — APPROVED** [review]
My christmas tree approves

**Review by anisometropie — APPROVED** [review]
My christmas tree approves

**Pull request merged**